pkg/daemon/daemon.go

```diff
@@ -492,6 +504,27 @@ func (d *UnikDaemon) initialize() {
 				mountPoints = strings.Split(mntStr, ",")
 			}
 
+			sizeStr := req.FormValue("size")
```

Comment:
See comment above - we can avoid bubbling by introducing `image_size` attribute into manifest.yaml.

One problem with dynamic determination is when you e.g. run database inside unikernel. If unikernel is barely big enough to fit libraries in it, where would the content of the database go?